eldritch/events.py

```diff
@@ -2734,6 +2742,56 @@ def log(self, state):
     return f"[{self.character.name}] rerolled {len(self.reroll_indexes)} dice on their {ctype}"
 
 
+class RerollSpecificDice(Event):
```

Comment:
Looking more closely, I think RerollSpecific will work for DiceRoll events too, if the DiceRoll is updated to have a count_successes() method.